infrastructure/smart-contracts/contracts/upgrades/node-license/NodeLicense8.sol

```diff
@@ -246,7 +246,7 @@ contract NodeLicense8 is ERC721EnumerableUpgradeable, AccessControlUpgradeable
         uint256 finalPrice = ethToXai(finalEthPrice);
 
         // Confirm the final price does not exceed the expected cost
-        require(_expectedCost >= finalPrice, "Price Exceeds Expected Cost");
```

Comment:
Was this not merged in fb-tiny-keys already ?

Reply:
It was in a different PR that had not been merged yet. I changed it here so the tests would pass. I can revert if you prefer.